src/main/java/org/codeforamerica/shiba/output/caf/HourlyJobIncomeInformation.java

```diff
@@ -1,18 +1,22 @@
 package org.codeforamerica.shiba.output.caf;
 
 import lombok.Value;
+import org.codeforamerica.shiba.Money;
 import org.codeforamerica.shiba.pages.data.Iteration;
 
+import java.math.BigDecimal;
+
 @Value
 public class HourlyJobIncomeInformation implements JobIncomeInformation {
-    Double hourlyWage;
-    Double hoursAWeek;
+    public static final BigDecimal WEEKS_IN_MONTH = new BigDecimal("4");
+    Money hourlyWage;
+    BigDecimal hoursAWeek;
     int indexInJobsSubworkflow;
     Iteration iteration;
 
     public HourlyJobIncomeInformation(String hourlyWage, String hoursAWeek, int indexInJobsSubworkflow, Iteration iteration) {
-        this.hourlyWage = hourlyWage.isEmpty() ? null : Double.valueOf(hourlyWage.replace(",",""));
-        this.hoursAWeek = hoursAWeek.isEmpty() ? null : Double.valueOf(hoursAWeek);
+        this.hourlyWage = hourlyWage.isEmpty() ? null : new Money(parseWithCommasRemoved(hourlyWage));
```

Comment:
Can we use $0 as the default here instead of null? And on the next line can we use 0 hours a week instead of null?

Reply:
It's used in Gross monthly income so it might show up as "0" instead of blank on the application if we set it to 0

Reply:
ah interesting. is that undesirable?

Reply:
I believe so

Reply:
Sree's right, is if we put a 0 that makes it appear they actually answered 0 which has implications for the application.